docs/pages/versions/unversioned/index.mdx

```diff
@@ -54,12 +54,12 @@ The easiest way to create a bare React Native app with support for the Expo SDK
 
 Every quarter there is a new Expo SDK release that typically updates to the latest stable version of React Native and includes a variety of bug fixes, features, and improvements to the Expo SDK.
```

Comment:
```suggestion
Every quarter there is a new Expo SDK release that typically updates to the latest stable versions of React Native and React Native Web, and includes a variety of bug fixes, features, and improvements to the Expo SDK.
```

As per our discussion yesterday on Slack, let's also include RNW in the text itself.